src/mainSection.js

```diff
@@ -0,0 +1,6 @@
+import htmlTag from './htmlTags';
```

Comment:
- [ ] You don't have to create the main section dynamically with javascript. Just add div with `id=content` inside body in the HTML document.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png)

Reply:
I preferred to create them in javascript just to be sure that everything was working, as practice to imporve my DOM manipulation skills, and i also think that is better for future improvements of the page